pages/partials/navbar.ejs

```diff
@@ -124,3 +125,18 @@ if (typeof authz_data !== 'undefined' && authz_data.has_instructor_view) {
 </ul>
 </div>
 </nav>
+<% // Mapping navPage to navtab sets
+const navPagesTabs = {
+    instance_admin: 'instructorInstanceAdminTabs',
+    course_admin: 'instructorCourseAdminTabs',
+    assessment: 'instructorAssessmentTabs',
+    question: 'instructorQuestionTabs'
+};
+// If navPage matches any of the keys in navPagesTabs, include the page given by its value
+if (typeof navPage != 'undefined' && Object.keys(navPagesTabs).includes(navPage)) { %>
```

Comment:
```suggestion
if (typeof navPage != 'undefined' && Object.hasOwnProperty(navPagesTabs, navPage)) { %>
```

It doesn't really matter here, but for large objects it's must faster to directly test whether they have a property (using `hasOwnProperty()`) rather than converting their keys to a list and doing an O(N) search with `includes()`.